bazel/glog.bzl

```diff
@@ -0,0 +1,165 @@
+# Implement a macro glog_library() that the BUILD file can load.
+# By default, glog is built with gflags support.  You can change this behavior by using
+# glog_library(with_gflags=0)
+#
+# This file is inspired by the following sample BUILD files:
+#       https://github.com/google/glog/issues/61
+#       https://github.com/google/glog/files/393474/BUILD.txt
+
+def glog_library(with_gflags=1):
+    native.cc_library(
+        name = 'glog',
+        visibility = [ '//visibility:public' ],
+        srcs = [
+            'src/base/commandlineflags.h',
+            'src/base/googleinit.h',
+            'src/demangle.cc',
+            'src/logging.cc',
+            'src/raw_logging.cc',
+            'src/signalhandler.cc',
+            'src/symbolize.cc',
+            'src/utilities.cc',
+            'src/vlog_is_on.cc',
+        ],
+        hdrs = [
+            'src/base/mutex.h',
+            'src/demangle.h',
+            'src/stacktrace.h',
+            'src/symbolize.h',
+            'src/utilities.h',
+            'src/glog/log_severity.h',
+        ],
+        includes = [
+            'src',
```

Comment:
This means that any users who use `#include "utilities.h"` to try to include their own `utilities.h` might accidentally pick up glog's `utilities.h`.

Reply:
Super good point here. This, and the comment below, will be addressed together.